### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by LOctavio**
For this branch, I created a table to store animals' information, inserted some data into it, and queried it.

Insert the following data:

- Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
- Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

Query the following:

- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+    id int,
```
Comment (AshakaE) [inline schema.sql]: - [OPTIONAL]
   - You can also set ID as primary key by `PRIMARY KEY (id)` and make it auto incremented with `id int NOT NULL IDENTITY(1,1)`

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+    name text,
```
Comment (AshakaE) [inline schema.sql]: - [x] Please add the `varchar` for name column

File: schema.sql [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+    date_of_birth datetime,
+    escape_attempts int,
+    neutered binary,
```
Comment (AshakaE) [inline schema.sql]: - [x] Please change column type to `boolean`

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name FROM animals WHERE neutered = true and escape_attempts<3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true; 
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
```
Comment (AshakaE) [inline queries.sql:9]: I believe what you're trying to say here is `WHERE name NOT  = gabumon` not the other way 💡

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @LOctavio  🙋‍♀️, 

Good job creating the pull request 👌

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 4 above)
Comment (LOctavio) [reply queries.sql:9]: Thanks. I tried what you suggested but there is a syntax error. 

I am using the syntax from [this](https://www.w3schools.com/sql/sql_and_or.asp) page and is working properly.

image

**Comment by Davidosky007** [comment]
## Hi @LOctavio 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**